core/src/main/java/org/apache/iceberg/rest/RESTSessionCatalog.java

```diff
@@ -366,6 +368,10 @@ public boolean tableExists(SessionContext context, TableIdentifier identifier) {
     }
   }
 
+  private Map<String, String> snapshotModeToParam(SnapshotMode mode) {
```

Comment:
nit: This method can be static. Alternatively, we could inline it since it's only used in one place.